dali/test/python/test_operator_warp.py

```diff
@@ -163,35 +163,39 @@ def test_cpu_vs_cv():
   for batch_size in [1, 4, 19]:
     for use_input in [False, True]:
       for (itype, otype) in io_types:
-        print("Testing cpu vs cv",
-              "\nbatch size: ", batch_size,
-              " matrix as input: ", use_input,
-              " input_type: ", itype,
-              " output_type: ", otype)
-        cv_pipeline = CVPipeline(batch_size, otype, itype, use_input);
-        cv_pipeline.build();
+        for inverted in [False, True]:
```

Comment:
I'd use some deterministic pseudorandom generator to get this value instead of doubling the number of test cases. This test is already quite slow.

Reply:
True that. It gets **really** slow when doubled.

Reply:
done